packages/web-components/src/components/background-media/background-media.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp.  2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html, property, customElement, LitElement, css } from 'lit-element';
+import settings from 'carbon-components/es/globals/js/settings';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
+import DDSImage from '../image/image';
+import styles from './background-media.scss';
+import { GRADIENT_DIRECTION, MOBILE_POSITION } from './defs';
+
+const { prefix } = settings;
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * Background Media.
+ *
+ * @element dds-background-media
+ * @slot long-description - The long description content.
+ * @slot icon - The icon content.
+ */
+@customElement(`${ddsPrefix}-background-media`)
+class DDSBackgroundMedia extends DDSImage {
+
+  @property({ attribute: 'source' })
+  source = ''; 
+
+  @property({ attribute: 'gradient-direction', reflect: true })
+  gradient_direction = GRADIENT_DIRECTION.RIGHT;
+
+  @property({ attribute: 'mobile-position', reflect: true })
+  mobile_position = MOBILE_POSITION.TOP;
+
+  render() {
+    
+    return html`
+      <div class="${prefix}--background-media">
+        <div class="${prefix}--background-media-container">
```

Comment:
Feels like some of these classes (eg. `${prefix}--background-media-container`) should be adjusted to be an element class name (eg. `${prefix}--background-media__container`) to follow the BEM structure. @ariellalgilmore Let me know if you agree.

Reply:
just messaged Dennis in slack. Agree to follow BEM structure, but hoping we can remove the `div`'s and use the `DDSImage` render() and handle most the functionality within the styles by using the different property values for gradient and mobile position